src/core/Pricing/PricingCards.tsx

```diff
@@ -204,15 +198,13 @@ const PricingCards = ({ data, delimiter }: PricingCardsProps) => {
                     </div>
                   )}
                 </div>
-                <div className="flex-1 flex flex-col gap-6 relative z-10">
-                  {sections.map(({ title, items, listItemColors, cta }) => (
-                    <div key={title} className="flex flex-col gap-3">
-                      <p className="text-neutral-800 dark:text-neutral-500 font-mono uppercase text-overline2 tracking-[0.16em]">
+                <div className="flex flex-col gap-4 relative z-10 flex-grow">
+                  {sections.map(({ title, items, listItemColors }) => (
+                    <div key={title} className="flex flex-col gap-2">
```

Comment:
Not sure if there's another reason for the change but gap-2 seems too small, the design has a 12px gap which maps to the original gap-3